server/routes/courtRegister/addNewCourtDetailsValidator.test.ts

```diff
@@ -44,13 +44,17 @@ describe('addNewCourtDetailsValidator', () => {
       const form = { ...validForm, id: 'A' }
       const nextPage = await validate(form, req, lookup)
       expect(nextPage).toEqual('/court-register/add-new-court-details')
-      expect(req.flash).toBeCalledWith('errors', [{ href: '#id', text: 'Court code must be at least 2 characters' }])
+      expect(req.flash).toBeCalledWith('errors', [
+        { href: '#id', text: 'Enter a court code between 2 and 12 characters' },
+      ])
```

Comment:
decided that GDS advice about reducing error message variations (which works well with the validator)